test/unit/test_runner.py

```diff
@@ -31,16 +33,6 @@ def rc(request, tmp_path):
     return rc
 
 
-@pytest.fixture(autouse=True)
-def mock_sleep(request, mocker):
-    # the handle_termination process teardown mechanism uses `time.sleep` to
-    # wait on processes to respond to SIGTERM; these are tests and don't care
-    # about being nice
-    m = mocker.patch('time.sleep')
-    m.start()
-    request.addfinalizer(m.stop)
```

Comment:
I checked the timing, and the whole module takes about 3.6 seconds. Maybe this was reducing it to 3.5 or 3.4, but the difference also could have been indistinguishable. I'm using `time.sleep` to make the assertion reliable, so that's why I did this.